Azure/sonic-buildimage
# tagged vlan read by minigraph

**- Why I did it**
To enable tagged vlan support in SONiC

**- How I did it**
Made a minor change to minigraph parser to distinguish between tagged and untagged vlans

**- How to verify it**
Test minigraph parser on a TOR by doing a reload of a  minigraph containing tagged vlans and ensure the correct configuration of the vlans on the device

**- Description for the changelog**
 


**- A picture of a cute animal (not mandatory but encouraged)**
